core/src/main/java/io/confluent/kafka/schemaregistry/rest/SchemaRegistryConfig.java

```diff
@@ -34,9 +35,6 @@
   // TODO: change this to "http://0.0.0.0:8081" when PORT_CONFIG is deleted.
   private static final String SCHEMAREGISTRY_LISTENERS_DEFAULT = "";
 
-  public static final String KAFKASTORE_SECURITY_PROTOCOL_SSL = "SSL";
```

Comment:
Hmm, will this have been released already with 3.0.1. We might need to keep these for compatibility since they are public. Mark as deprecated instead?